### PR TITLE
Update Granular Access page with supported resources

### PR DESCRIPTION
**Pull request by saidurgas**
### What does this PR do? What is the motivation?
Update Granular access page with supported resources

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by Jennyliu00 — APPROVED** [review]

File: content/en/account_management/rbac/granular_access.md [hunk 1]
```diff
@@ -14,12 +14,16 @@ Use the different principals to control access patterns in your organization and
 
 | Supported resources with granular access control | Team-based access | Role-based access | User / service account-based access |
 |--------------------------------------------------|-------------------|-------------------|-------------------------------------|
+| [AppBuilder] [8]                                 | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
```
Comment (aliciascott) [inline content/en/account_management/rbac/granular_access.md:17]:
```suggestion
| [AppBuilder][8]                                 | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @saidurgas the links are broken as there is an extra space, can you fix those and request re-review then I'll merge it !

**Comment by sanjaythakkar-dd** [comment]
looks like [modified files](https://docs-staging.datadoghq.com/saidurga.subramanian/update_granular_access_page/account_management/rbac/granular_access) doesnt have correct formatting

**Review by aliciascott — APPROVED** [review]
looks good thanks for fixing the links! @saidurgas is this ready to be merged as well ?

**Comment by sanjaythakkar-dd** [comment]
overall looks good
again same comment as https://github.com/DataDog/datadog-api-spec/pull/2763#discussion_r1572680050
my thought here is
if workflows give 👍  feel free to merge it
if not, maybe remove workflows for now and merge it

**Comment by saidurgas** [comment]
> looks good thanks for fixing the links! @saidurgas is this ready to be merged as well ?

yes ready to be merged

**Pull request merged**